indexer/feature.hpp

```diff
@@ -41,12 +40,12 @@ class FeatureBase
 
   feature::EGeomType GetFeatureType() const;
 
-  inline uint8_t GetTypesCount() const
+  uint8_t GetTypesCount() const
```

Comment:
`inline` only make noise in this context. Functions defined in class scope are inline by default.